src/librustdoc/html/render/print_item.rs

```diff
@@ -1415,3 +1423,42 @@ fn document_non_exhaustive(w: &mut Buffer, item: &clean::Item) {
         w.write_str("</div>");
     }
 }
+
+fn document_ty_layout(w: &mut Buffer, cx: &Context<'_>, ty_def_id: DefId) {
+    if !cx.shared.show_type_layout {
+        return;
+    }
+
+    let param_env = cx.tcx().param_env(ty_def_id);
+    let ty = cx.tcx().type_of(ty_def_id);
+    match cx.tcx().layout_of(param_env.and(ty)) {
+        Ok(ty_layout) => {
+            writeln!(w, "<h2 class=\"small-section-header\">Layout</h2>");
+            writeln!(w, "<div class=\"docblock\">");
```

Comment:
Adding `docblock` here is a bit of hack. Without `docblock`, links are rendered the same as the surrounding text, which means users will likely not notice the link to the Reference. Unfortunately, `docblock` also adds a toggle to hide the docs, but 🤷 

If you have ideas for a less hacky approaches, I'm all ears!

Reply:
Using `docblock` also means that inline code is rendered the same as the content of the docs.

Reply:
Can you post a picture of what it looks like without `docblock`? I'm not a fan of adding more toggles.

Reply:
I think the goal is to get rid of the dynamically-inserted toggles, so once that happens this won't have a toggle. It looks like this without `docblock`:

![image](https://user-images.githubusercontent.com/37223377/114606525-38d91680-9c50-11eb-98aa-41cc401a0f4b.png)

You can't see the link at all, and the inline code styling looks different from the rest of the docs.

Reply:
@GuillaumeGomez do you have suggestions for rendering this with the correct highlighting but without a toggle?

Reply:
Try adding this rule for the link: `.small-section-header + div a {` (alongside where the `.docblock a` rule is).

Reply:
Shouldn't we hide this section by default? Or maybe we just keep it this way?

Reply:
> Try adding this rule for the link: `.small-section-header + div a {` (alongside where the `.docblock a` rule is).

Hmm, that causes other styles to change:

![image](https://user-images.githubusercontent.com/37223377/115931675-6720d800-a440-11eb-9092-7acd18e1fd47.png)

Maybe we should just leave it as docblock for now?

Reply:
I could also try adding a new `.info-block` (or something like that) class, but there might be other styles that we want to share between it and `.docblock`.

Reply:
Instead of doing that, can't you add an extra class and then, if it has this class, the JS doesn't add a toggle?

Reply:
I'll try that. Hopefully the work to convert the toggles to HTML should mean this is a temporary fix.

Reply:
Okay, that seems to work!

![image](https://user-images.githubusercontent.com/37223377/116307582-5b445700-a75b-11eb-86c2-68967c93c0f8.png)